pkg/services/dashboards/database/database.go

```diff
@@ -670,8 +670,14 @@ func (d *dashboardStore) RestoreDashboard(ctx context.Context, dashboardUID stri
 
 func (d *dashboardStore) SoftDeleteDashboard(ctx context.Context, dashboardUID string) error {
 	return d.store.WithTransactionalDbSession(ctx, func(sess *db.Session) error {
-		sql := "UPDATE dashboard SET deleted=? WHERE uid=?"
-		_, err := sess.Exec(sql, time.Now(), dashboardUID)
+		_, err := sess.Exec("UPDATE dashboard SET deleted=?, folder_id=0, folder_uid=NULL WHERE uid=?", time.Now(), dashboardUID)
+		return err
+	})
+}
+
+func (d *dashboardStore) SoftDeleteDashboardsInFolder(ctx context.Context, folderUid string) error {
+	return d.store.WithTransactionalDbSession(ctx, func(sess *db.Session) error {
```

Comment:
same question here

Reply:
In this case, it could make sense because we don't want to soft-delete a folder